blueprints/app/files/.eslintrc.js

```diff
@@ -2,12 +2,39 @@ module.exports = {
   root: true,
   parserOptions: {
     ecmaVersion: 2017,
-    sourceType: 'module'
+    sourceType: 'module',
```

Comment:
seems strange to only have trailing commas in this file. I vote for removal.

Reply:
😭 but ok 😄

Reply:
I agree, we should add them to all files...

Reply:
Ya, agreed. Though I’m happy to do this separately.